### ISSUE TITLE
bignutty.xyz

### ISSUE THREAD

**Issue by dekatr0n** [easylist/easylist#16871]
#### List the website(s) you're having issues with:
```
bignutty.xyz
very.bignutty.xyz
```

#### What happens?
The domain is blocked by easylist (easylist_adservers.txt), but the domain is not an ad server.

#### Filter lists you're using:

EasyList

#### Your settings:

- OS/version: Windows 10
- Browser/version: Firefox 115
- Adblock extension/version: uBlock Origin 1.51.0

#### Other details:

Seems like it was added in this commit: https://github.com/easylist/easylist/commit/35193b008493490d66f60f38d1c7af8f6def0508

**Comment by ryanbr** [comment]
So what is the server used for?

**Comment by dekatr0n** [comment]
I'm not the server owner, but `very.bignutty.xyz` is a [Firefish](https://joinfirefish.org/) instance that I've been following for fediverse updates. The main domain `bignutty.xyz` seems to be an admin panel for a seedbox. `meowlytics.bignutty.xyz` is hosting a Plausible instance possibly for analytics for the domain.

I opened this issue because I thought the domain was mistakenly added to EasyList. Although I admit I cannot vouch for the entire server. I tried looking, but is there any issue that indicates why the domain was added?
